build.sbt

```
@@ -58,145 +54,61 @@ val coreDeps = Seq(
+ThisBuild / githubWorkflowJavaVersions := Seq("11", "17").map(JavaSpec.temurin(_))
+ThisBuild / tlCiReleaseBranches := Seq("series/0.7")
+ThisBuild / tlSitePublishBranch := Some("series/0.7")
 
 lazy val docsSettings =
   Seq(
-    mdocIn := (baseDirectory.value) / "src" / "main" / "mdoc", //
-    mdocVariables := Map(
-      "VERSION" -> version.value,
-      "BINARY_VERSION" -> binaryVersion(version.value),
+    fork := true,
+    mdocVariables ++= Map(
       "HTTP4S_VERSION" -> http4sV,
       "HTTP4S_VERSION_SHORT" -> http4sV.split("\\.").take(2).mkString("."),
       "SCALA_VERSION" -> CrossVersion.binaryScalaVersion(scalaVersion.value),
       "SCALA_VERSIONS" -> formatCrossScalaVersions((core / crossScalaVersions).value.toList)
     ),
-    generateNetlifyToml := {
-      val toml = latestStableVersion(baseDirectory.value)
-        .map(v => s"""
-           |[[redirects]]
-           |  from = "/stable/*"
-           |  to = "/$v/:splat"
-           |  force = false
-           |  status = 200
-           |""".stripMargin)
-        .getOrElse("") + s"""
-           |[[redirects]]
-           |  from = "/*"
-           |  to = "/latest/:splat"
-           |  force = false
-           |  status = 302
-           |""".stripMargin
-      IO.write(target.value / "netlify.toml", toml)
-    },
-    Compile / paradox / sourceDirectory := mdocOut.value,
-    makeSite := makeSite.dependsOn(mdoc.toTask("")).dependsOn(generateNetlifyToml).value,
-    ghpagesPushSite := (ghpagesPushSite.dependsOn(makeSite)).value,
-    Compile / paradoxMaterialTheme ~= {
-      _.withRepository(uri("https://github.com/http4s/http4s-jdk-http-client"))
-        .withLogoUri(uri("https://http4s.org/images/http4s-logo.svg"))
-    },
-    git.remoteRepo := "git@github.com:http4s/http4s-jdk-http-client.git",
-    Paradox / siteSubdirName := {
-      if (isSnapshot.value) "latest"
-      else version.value
-    },
-    makeSite / mappings ++= Seq(
-      target.value / "netlify.toml" -> "netlify.toml"
+    tlSitePublish := List( // TODO remove after upstreamed
+      WorkflowStep.Use(
+        UseRef.Public("peaceiris", "actions-gh-pages", "v3.8.0"),
+        Map(
+          "github_token" -> s"$${{ secrets.GITHUB_TOKEN }}",
+          "publish_dir" -> (ThisBuild / baseDirectory).value.toPath.toAbsolutePath
+            .relativize(((Laika / target).value / "site").toPath)
+            .toString,
+          "keep_files" -> "true"
+        )
+      )
     ),
-    ghpagesCleanSite / includeFilter :=
-      new FileFilter {
-        def accept(f: File) =
-          f.toPath.startsWith(
-            (ghpagesRepository.value / (Paradox / siteSubdirName).value).toPath
-          )
-      } || "netlify.toml"
+    unusedCompileDependenciesFilter -= moduleFilter("org.http4s", "http4s-blaze-server"),
+    unusedCompileDependenciesFilter -= moduleFilter("org.http4s", "http4s-dsl"),
+    unusedCompileDependenciesFilter -= moduleFilter("org.scalameta", "mdoc")
```

Maybe we just want a catch all?
```suggestion
    unusedCompileDependenciesFilter -= moduleFilter()
```

---

Cool, don't have much experience with this plugin 👍